### ISSUE TITLE
Enable the debug button for everyone

### ISSUE THREAD

**Issue by Bluscream** [dehesselle/dayzsrvip#8]
I see no sense in locking this. I had to use my [unlocker](https://github.com/Bluscream/pyTSon_plugins/blob/master/scripts/unlocker.py) to view the debug window once.

If smeone doesn't feel like a dev he just doesn't click it :)

**Comment by dehesselle** [comment]
Honestly, I was rather thinking about removing it altogether at some point, hoping it becomes useless. The debug-functionality has been there from the start/to help me getting started so I could inject messages into the system without having to build the `.DLL` and run it in TeamSpeak. It was even more hidden (read: invisible) at first, but I became uncomfortable with the way I did that. So I decided to turn this into a (in release builds disabled) button until I decided to drop it.

Since we will be moving off of sending text messages, I guess this debug-functionality will become valuable again for testing purposes - no other way of inserting messages (read: commands) into the system. I'll put in a warning though, like the one you get when you use `about:config` in Firefox ("you have been warned"), and then all bets are off - if someone wants to mess with this, so be it. (Of course I'll have to put in safeguards/input checking to not endanger the system.)
